addons/stock/models/stock_move_line.py

```diff
@@ -112,11 +112,18 @@ def _check_positive_qty_done(self):
         if any(ml.qty_done < 0 for ml in self):
             raise ValidationError(_('You can not enter negative quantities.'))
 
+    @api.onchange('result_package_id')
+    def _onchange_result_package_id(self):
+        if self.result_package_id:
+            if not self.id and self.user_has_groups('stock.group_stock_multi_locations'):
```

Comment:
group_stock_storage_categories

Reply:
I don't think we should change to `group_stock_storage_categories`. We allow to use putaway rules for packages without storage categories.

Reply:
If the group is not active all putaways are archived, right ?

Reply:
if no `group_stock_storage_categories`, only putaways with  `storage_category_id` are archived

Reply:
You're right. my bad :disappointed: